docs/api.md

```diff
@@ -115,4 +142,22 @@ function do_not_send_vary_headers() : array {
 		'Interest' => false,
 	];
 }
+```
+
+### `pantheon.ei.custom_header_data`
+
+Modify HeaderData being set as a vary header from update_vary_headers.
```

Comment:
Since `HeaderData` is the class name, it should be wrapped in backticks. Also the function name. 🔢

```suggestion
Modify `HeaderData` being set as a vary header from `update_vary_headers`.
```